uint8 frame
geometry_msgs/Pose2D[] targetTranslation
float64[] targetRotation
float64 step
bool offsetTraslation
bool offsetRotation
